.github/workflows/daily-test-build.yml

```diff
@@ -1,13 +1,31 @@
 name: Daily Test Build TileDB-Py Against Core
 
-#on: [push]
+on: [push]
 
-on:
-  schedule:
-    # runs every day at 5:00 UTC (1:00AM EST / Midnight CST)
-    - cron: "0 5 * * *"
+# on:
+#   schedule:
+#     # runs every day at 5:00 UTC (1:00AM EST / Midnight CST)
+#     - cron: "0 5 * * *"
 
 jobs:
+  test-wheels-on-azure:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Get current date
+      id: date
+      run: echo "::set-output name=date::$(date +'%a-%Y-%m-%d')"
+    - name: Create Test Branch for Azure Wheel Nightly Build
+      uses: peterjgrainger/action-create-branch@v2.0.1
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        branch: 'azure-wheel-test-${{ steps.date.outputs.date }}'
+    - name: Clean Up Test Branch
```

Comment:
Maybe this part should run un-conditionally on the repo trigger event? Just wondering if there is any risk of a race condition where the branch has been deleted before the azp job starts.